### PR TITLE
131 entry fields

### PR DESCRIPTION
**Pull request by sensei100**
Separated out the logic for preceding and succeeding entries based on indicator 2

### CODE REVIEW

File: app/controllers/catalog_controller.rb [hunk 1]
```diff
@@ -167,7 +167,7 @@ class CatalogController < ApplicationController
     config.add_index_field 'imprint', label: 'Published'
     config.add_index_field 'creator', label: 'Author/creator'
     config.add_index_field 'format', label: 'Resource Type'
-
+    config.add_index_field 'entry_preced', label: 'Entry Preced'
```
Comment (bibliotechy) [inline app/controllers/catalog_controller.rb]: @sensei100 Is 'Entry Preced' a typo?

**Comment by sensei100** [comment]
No, that is how it is written in all the documentation.
https://tulibdev.atlassian.net/wiki/spaces/SAD/pages/22839300/Data+Displays#DataDisplays-FieldstoincludeinRecordPages

**Comment by bibliotechy** [comment]
@ebtoner Any thoughts on this? Is "Entry Preced" a typo?

**Comment by bibliotechy** [comment]
@sensei100 did you get an update on this from Emily?

**Comment by sensei100** [comment]
Superseded by #50

**Pull request closed without merging**